Library/Homebrew/Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-if ENV.fetch("HOMEBREW_DEVELOPER", "").empty? || ENV.fetch("HOMEBREW_USE_RUBY_FROM_PATH", "").empty?
```

Comment:
It still is for using Ruby - this is just gem installation and this env check only exists really for Dependabot, otherwise I would just do `>= 2.6.0`.

The reason for dropping the developer bit it so we don't need to add that to the `brew tests` whitelist, which will likely cause issues.

Reply:
Ok, fine with me 👍🏻